manifests/03-cluster-operator.yaml

```diff
@@ -3,3 +3,7 @@ kind: ClusterOperator
 metadata:
   name: ingress
 spec: {}
+status:
+  versions:
+    - name: operator
+      version: "0.0.1-snapshot"
```

Comment:
Is this intentional? I don't understand why it's hard-coded here and not empty until updated by the operator

Reply:
fallout from copy+paste!, these values are no longer hard-coded.